Symfony/CS/Tests/Fixer/PSR2/MultipleUseFixerTest.php

```diff
@@ -40,21 +40,22 @@ public function provideCases()
             ),
             array(
                 <<<'EOF'
-use Some, Not, PHP, Like, Use, Statement;
```

Comment:
https://3v4l.org/OR5lX

"Fatal error: Namespace declaration statement has to be the very first statement or after any declare call in the script in /in/OR5lX on line 15"